tools/src/main/scala/scala/scalanative/compiler/tools/DispatchInfoParser.scala

```diff
@@ -0,0 +1,42 @@
+package scala.scalanative
+package compiler
+package tools
+
+import scala.util.parsing.combinator._
+import scala.util.matching.Regex
+
+object DispatchInfoParser extends JavaTokenParsers {
```

Comment:
Scala's parser combinators are not being actively maintained. Using `fastparse` for this kind of things is probably a better idea.